app/src/main/java/com/github/alfabravo2013/readyforexams/util/Utils.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.alfabravo2013.readyforexams.util
+
+import android.util.Patterns
+
+const val MIN_PASSWORD_LENGTH = 9
+const val MAX_PASSWORD_LENGTH = 20
+
+fun String.isInvalidEmail(): Boolean = Patterns.EMAIL_ADDRESS.matcher(this).matches().not()
+
+fun String.isInvalidPassword(): Boolean = length !in MIN_PASSWORD_LENGTH..MAX_PASSWORD_LENGTH
+
+open class OneTimeEvent<out T>(private val content: T) {
```

Comment:
You can use a SingleLiveEvent: https://github.com/android/architecture-samples/blob/dev-todo-mvvm-live/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/SingleLiveEvent.java or maybe rename your class to SingleEvent or something?